src/explorer.ts

```diff
@@ -20,13 +20,28 @@ export function createKubernetesResource(kind: kuberesources.ResourceKind, id: s
 }
 
 function getIconForPodStatus(status: string): vscode.Uri {
-    if (status === "running" || status === "completed") {
+    if (status === "Running" || status === "Completed") {
         return vscode.Uri.file(path.join(__dirname, "../../images/runningPod.svg"));
     } else {
         return vscode.Uri.file(path.join(__dirname, "../../images/errorPod.svg"));
     }
 }
 
+function parsePodLine(podInfo: string[]): any[] {
+    const headers = podInfo.shift();
+    const parsedHeaders = headers.toLowerCase().replace(/\s+/g, '|').split('|');
+    const pods = [];
+    podInfo.map((line) => {
```

Comment:
This construct maps a `void` function over the string array, and uses the side effect of the `void` function pushing an object onto an object array.  Prefer `const pods = podInfo.map((line) => { ...; return podObject; });` - this says to map an object-returning function over the string array to produce an array of objects directly.

Reply:
perfect, will change this.